src/components/Personalization/createCollect.js

```diff
@@ -13,7 +13,7 @@ governing permissions and limitations under the License.
 export default ({ eventManager, mergeMeta }) => {
   return meta => {
     const event = eventManager.createEvent();
-
+    event.mergeXdm({ eventType: "display" });
```

Comment:
We should discuss adding an eventType to the known list in xdm, here: https://github.com/adobe/xdm/blob/b092f1c71966d1c42ca59f8f863379fdaf033bc5/schemas/data/time-series.schema.json#L35

Reply:
We have discussed the `eventType` and we know that we don't have yet XDM mixins for it. We'll go with some defaults and work on XDM mixins later on.

Reply:
@jfkhoury @dancingcactus